packages/release-mastermind/src/utils/labels.ts

```diff
@@ -6,80 +1,88 @@
 
 /**
  * Syncronise labels to repository
  * @author IvanFon, TGTGamer, jbinda
  * @since 1.0.0
  */
-export async function sync(this: Utils, config: Runners['labels']) {
-  /**
-   * Syncronises the repo labels
-   * !todo Add delete labels
-   * @since 2.0.0
-   */
-  if (!config) throw new Error('Cannot syncronise labels without config')
-  const curLabels: Labels = await this.api.labels.get().catch(err => {
-    throw log(
-      LoggingLevels.error, `Error thrown while getting labels: ` + err)
-  })
-  log(LoggingLevels.debug, `curLabels: ${JSON.stringify(curLabels)}`)
-  for (const configLabel of Object.values(config)) {
-    const label = curLabels[configLabel.name.toLowerCase()]
+export async function sync(this: Utils, config: Runners["labels"]) {
+	/**
+	 * Syncronises the repo labels
+	 * !todo Add delete labels
+	 * @since 2.0.0
+	 */
+	if (!config) throw new Error("Cannot syncronise labels without config")
+	const curLabels: Labels = await this.api.labels.get().catch((err) => {
+		throw log(LoggingLevels.error, `Error thrown while getting labels: ` + err)
+	})
+	log(LoggingLevels.debug, `curLabels: ${JSON.stringify(curLabels)}`)
+	for (const configLabel of Object.values(config)) {
+		const label = curLabels[configLabel.name.toLowerCase()]
 
-    /**
-     * Update label
-     * @author IvanFon, TGTGamer, jbinda
-     * @since 1.0.0
-     */
-    if (label) {
-      if (
-        label.name !== configLabel.name ||
-        (label.description !== configLabel.description &&
-          configLabel.description !== undefined) ||
-        label.color !== formatColor(configLabel.color)
-      ) {
-        log(
-            LoggingLevels.info,
-            `Recreate ${JSON.stringify(configLabel)} (prev: ${JSON.stringify(
-              label
-            )})`
-        )
-        log(LoggingLevels.debug, `current name: ${label.name.toString()} | new: ${configLabel.name.toString()}`)
-        await this.api.labels.update(label.name, configLabel).catch(err => {
-          log(
-            LoggingLevels.error, `Error thrown while updating label: ` + err)
-        })
-      } else {
-        log(
-          LoggingLevels.info,
-            `No action required to update label: ${label.name}`
-        )
-      }
+		/**
+		 * Update label
+		 * @author IvanFon, TGTGamer, jbinda
+		 * @since 1.0.0
+		 */
+		if (label) {
+			if (
+				label.name !== configLabel.name ||
+				(label.description !== configLabel.description &&
+					configLabel.description !== undefined) ||
+				label.color !== formatColor(configLabel.color)
+			) {
+				log(
+					LoggingLevels.info,
+					`Recreate ${JSON.stringify(configLabel)} (prev: ${JSON.stringify(
+						label
+					)})`
+				)
+				log(
+					LoggingLevels.debug,
+					`current name: ${label.name.toString()} | new: ${configLabel.name.toString()}`
+				)
+				await this.api.labels.update(label.name, configLabel).catch((err) => {
+					log(LoggingLevels.error, `Error thrown while updating label: ` + err)
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Videndum/action-masterminds/pullRequest?prid=7872005)
```suggestion
          log(LoggingLevels.error, "Error thrown while updating label: " + err)
```